spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -764,7 +770,11 @@ application_response_time_seconds{quantile="0.999"} 0.005608694
 # HELP application_response_time_total The number of calls to this REST endpoint #(1)
 application_response_time_total 12
 # TYPE application_response_time_elapsedTime_seconds gauge
-application_response_time_elapsedTime_seconds  12.3
+application_response_time_elapsedTime_seconds  12.30000000
+# TYPE application_response_time_maxTimeDuration_seconds gauge
+application_response_time_maxTimeDuration_seconds  3.231000000
+# TYPE application_response_time_minTimeDuration_seconds gauge
+application_response_time_minTimeDuration_seconds  0.0256
```

Comment:
There should be a note somewhere around here that it will be `NaN` if no measurements are available

Reply:
note added